examples/nginx-grafana-metabase/README.md

```diff
@@ -0,0 +1,162 @@
+179.43.177.xxx - - [26/Jun/2023:06:45:28 +0000] "GET / HTTP/1.1" 301 60 "-" "Hello World"
+87.236.176.xxx - - [26/Jun/2023:07:58:36 +0000] "GET / HTTP/1.1" 200 6364 "https://www.ayewo.com/" "Mozilla/5.0 (compatible; InternetMeasurement/1.0; +https://internet-measurement.com/)"
+87.236.176.xxx - - [26/Jun/2023:07:58:38 +0000] "GET /favicon.png HTTP/2.0" 200 7636 "-" "Mozilla/5.0 (compatible; InternetMeasurement/1.0; +https://internet-measurement.com/)"
+93.126.72.xxx - - [26/Jun/2023:08:29:38 +0000] "GET /how-to-install-rsync-on-windows/ HTTP/2.0" 200 6394 "https://www.google.com/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
+```
+
+The first line in the access log (from the IP address `161.35.230.x`) is actually a maliciously crafted request that doesn't even specify a HTTP method (i.e. `GET` or `OPTIONS`) to the server which is why the server responded with a HTTP 400 code (Bad Request). 
+
+There are several of such malformed requests in the access logs that are tricky to parse which is why an additional column `malicious_request` was added to the `nginx_historical_access_log` stream which was created in the previous article.
+
+### Nginx Access Log Format
+Nginx's [access log format](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#setting-up-the-access-log) is specified in `/etc/nginx/nginx.conf` and it broadly looks like this:
+```bash
+http {
+    log_format compression '$remote_addr - $remote_user [$time_local] '
+                           '"$request" $status $body_bytes_sent '
+                           '"$http_referer" "$http_user_agent"';
+
+    server {
+        access_log /var/log/nginx/access.log;
+        ...
+    }
+}
+```
+
+Below is a brief explanation of how each access log line is parsed into 12 columns:
+* `$remote_addr` is stored in column `remote_ip` of type `ipv4`;
+* `-` is stored in column `rfc1413_ident` of type `string`;
+* `$remote_user` is stored in column `remote_user` of type `string`;
+* `$time_local` os stored in column `date_time` of type `datetime64`;
+* `$request` is split into 4 columns each of type `string` to make our analysis easier:
+  * column `http_verb` of type `string`;
+  * column `path` of type `string` and;
+  * column `http_ver` of type `string`;
+  * column `malicious_request` of type `string` for any `$request` that fails to parse correctly; 
+* `$status` is stored in column `status` of type `int`;
+* `$body_bytes_sent` is stored in column `size` of type `int`;
+* `$http_referer` is stored in column `referer` of type `string`;
+* `$http_user_agent` is stored in column `user_agent` of type `string`.
+
+The previous article created a stream with a total of 12 columns to take advantage of Timeplus Proton's fast aggregation for our analysis. We will reuse the same approach in this article. 
```

Comment:
Now that you talk about column, maybe have a table in Markdown to show such 12 columns, with column name, data type, meaning and sample value